### PR TITLE
[LL-4653] Redirect outdated app to CTA manager

### PR DESCRIPTION
**Pull request by LFBarreto**
[LL-4653] Redirect outdated app to CTA manager
### Type

UI Polish
### Context

[LL-4653]
### Parts of the app affected / Test plan

[LL-4653]: https://ledgerhq.atlassian.net/browse/LL-4653
[LL-4653]: https://ledgerhq.atlassian.net/browse/LL-4653

### CODE REVIEW

**Comment by LFBarreto** [comment]
copy of #3643 without conflicts

File: src/renderer/components/DeviceAction/rendering.js [hunk 1]
```diff
@@ -384,24 +384,29 @@ export const renderError = ({
...
-      {onRetry ? (
-        <Button primary ml={withExportLogs ? 4 : 0} onClick={onRetry}>
-          <Trans i18nKey="common.retry" />
-        </Button>
-      ) : null}
+      {managerAppName ? (
+        <OpenManagerButton appName={managerAppName} />
+      ) : (
+        <>
+          {supportLink ? (
```
Comment (JunichiSugiura) [inline src/renderer/components/DeviceAction/rendering.js:391]: `supportLink && <Component />` would be enough.

**Review by JunichiSugiura — DISMISSED** [review]
LGTM

**Comment by Arnaud97234** [comment]
@LFBarreto The CTA that should redirect to the manager is missing.
Use case:
- Polkadot app is outdated (You can find the NanoX version on provider 4)
- In the add account flow, when opening Polkadot app you see the outdated app Modal.
- On this modal there is no CTA redirecting to the Manager

File: src/renderer/components/DeviceAction/index.js [hunk 2]
```diff
@@ -190,11 +190,22 @@ const DeviceAction = <R, H, P>({
   }
 
   if (!isLoading && error) {
-    if (error instanceof ManagerNotEnoughSpaceError || error instanceof OutdatedApp) {
-      return renderError({ error, withOpenManager: true });
+    if (
+      error instanceof ManagerNotEnoughSpaceError ||
+      error instanceof OutdatedApp ||
+      error.name === "UpdateYourApp"
```
Comment (gre) [inline src/renderer/components/DeviceAction/index.js:196]: it should be `error instanceof UpdateYourApp` like done for others 🤔

**Review by gre — CHANGES_REQUESTED** [review]

**Review by machard — APPROVED** [review]

**Review by Arnaud97234 — DISMISSED** [review]
If manager app update is mandatory then user see an error modal with a CTA that redirect to the manager.

@machard Some screenshots needs to be updated

✅

**Comment by machard** [comment]
@Arnaud97234 I can not push on @LFBarreto branch

**Comment by gre** [comment]
@LFBarreto unclear status on this one.

**Review by gre — APPROVED** [review]

**Comment by gre** [comment]
@Arnaud97234 the two last commit of @LFBarreto to me require a new QA on this. Even tho you did accept it, there are changes that I think invalidate your review. I let you both work on this. thanks.

**Review by ychen-ledger — APPROVED** [review]
Validated with: 

- MacOS Big Sur Version 11.4
- Polkadot (DOT) Version 5.23.0
- FW version :1.2.4-6
- Manger provider 4
- Portfolio, Accounts, Send and Receive

**Pull request merged**